app/policies/application_policy.rb

```diff
@@ -19,6 +19,18 @@ def superadmin_or_volunteer?
     superadmin? || volunteer?
   end
 
+  def superadmin_or_department_manager?
+    superadmin? || department_manager?
+  end
+
+  def departments_offer?
+    department_manager? && record.department == user.department.first
```

Comment:
What if `user.department.first` doesn't match, but `user.department.last` would?
Or is that impossible?

Reply:
Or in Other words, wouldn't it be safer to do:
```ruby
user.department.include? record.department
```
?